### DISCUSSION TITLE
Audio Settings: Choose Audio Input/Output

### DISCUSSION THREAD

**Discussion by AquaSpider** [stoatchat/discussions#66] [Feature Suggestions]
Some people use a microphone or headphone channel that is different from their default.  It appears that Revolt's is currently tied to the default, or whatever chrome is using.  By being able to choose the input and output sources, it makes Revolt much more versatile for those with a lot of hardware connected to a single computer, and helps it compete better with other chat software out there.  
Are there plans to put this in soon?  And if not, will you consider doing so?  

There's also the possibility I'm being dumb and the option is already in there somewhere.

Anyway, thanks for your time!  I greatly appreciate the work y'all do, and I'm glad you chose to take this project on.

**Comment by ryanalexander** [comment]
Already being implemented. https://github.com/revoltchat/revite/pull/116

Just waiting on @insertish to approve

**Reply by AquaSpider** [reply]
Yay!
